src/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,56 @@
+.alert {
+  display: flex;
+  width: 100%;
+  margin-bottom: 2rem;
+
+  animation: slide-in 0.2s ease-in;
+
+  border-radius: 12px;
+
+  background: rgba(37, 37, 37, 0.6);
+
+  color: #fff;
+  backdrop-filter: blur(27.1828px);
+  @keyframes slide-in {
+    0% {
+      transform: skewX(10deg) translateX(-25px);
+    }
+
+    100% {
+      transform: skewX(0deg);
+    }
+  }
+
+  .alert-icon {
+    position: absolute;
+    margin-top: 1.18rem;
+    margin-left: 1.18rem;
+  }
+}
+
+.alert-content {
+  ul {
+    padding-left: 4.68rem;
+
+    opacity: 0.6;
+
+    color: #fff;
+    font-family: 'Neuzeit S LT Std Nomal', sans-serif;
```

Comment:
You can specify the font-family as a variable at the top level (inside the _variables.scss file) so you don't have to keep redefining it in every class 👍 
https://getbootstrap.com/docs/5.0/content/typography/#global-settings

The full list of variables you can override can be found [here](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss)

Reply:
Or you can set the font-family for the `html` or `body` tag, it will apply to children tags

Reply:
I've created variables inside the _variables.scss because there are bold and normal style so I can use them depending on the design in figma. Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/f5c1469821f5cd51510a802528ec903b4dd6df28